src/app/components/job/action/add/job-manager-task-picker.html

```diff
@@ -28,6 +28,13 @@
             <bl-duration-picker formControlName="retentionTime" label="Retention time" [allowUnlimited]="false" [defaultDuration]="7" required></bl-duration-picker>
         </bl-form-field>
     </div>
+    <div class="form-element">
+        <bl-form-field>
+            <input blInput formControlName="requiredSlots" placeholder="Required slots">
+        </bl-form-field>
+        <bl-error controlName="requiredSlots" code="required">Required slots is a required field</bl-error>
+        <bl-error controlName="requiredSlots" code="min">Required slots value must be greater than 0</bl-error>
```

Comment:
Do we enforce only integer is specified for requiredSlots? We might want to make the description more accurate like "greater than or equal to 1".

Reply:
I agree this change would be better, I'll update that.